cypress/integration/pages/frontPage/tests.js

```diff
@@ -206,6 +223,36 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
             },
           );
         });
+
+        it('should contain Radio Bulletin if a promo of type RadioBulletin is in the feed', () => {
+          cy.request(`${config[service].pageTypes.frontPage.path}.json`).then(
```

Comment:
this bit is being repeated like 4 times in this file, and tbh it's a bit of an eyesore.. could this be used as a variable?

Reply:
Something like:
```
const getServiceJSON = service =>
  `${config[service].pageTypes.frontPage.path}.json`;
```
?

Reply:
yeah, I just think that looks cleaner

Reply:
Done!